README.md

```diff
@@ -0,0 +1,11 @@
+# github-workflow-commands
+
+For printing workflow commands in GitHub Actions.
+
+See [Workflow commands for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions).
+
+The code herein is based on [GitHub Actions Toolkit](https://github.com/actions/toolkit/tree/main/packages/core/src).
```

Comment:
Should we consider making an `actions-toolkit` library instead, that this is a part of? Being about to set outputs, add to env, read inputs, write a build summary, etc would all be useful if writing actions in Haskell.

Reply:
Maybe! I understand this particular scope a lot better than all of that, so I think I'd rather ship this now and deprecate it later if we take on the bigger project.

Reply:
Fair enough. Building an Action in Haskell is probably a (much) smaller use-case since the ergonomics (doing it as a Docker action) aren't great. It's annoying enough I tend to just jump to TypeScript. But just outputting in a format GHA understands is something that'd be more useful in more places.